harness/determined/horovod.py

```diff
@@ -68,23 +69,22 @@ def __getattr__(self, attr: str) -> Any:
 
     def cross_rank(self) -> Any:
         """
-        Horovod versions <= v0.21.3 accidentally did not expose cross_rank or cross_size for keras.
-
-        See https://github.com/horovod/horovod/pull/3008.
+        When hvd.cross_rank() is not reliably present (version =< v0.22.1) we fall back to reading
```

Comment:
```suggestion
        When hvd.cross_rank() is not present (version =< v0.22.1) we fall back to reading
```

I thought it was not present *at all* before 0.22.1? This word "reliably" gives me impression as if it is some non-deterministic behavior.

Reply:
No, `horovod.torch.cross_rank()` was available, as was `horovod.tensorflow.cross_rank()`, but there was a bug (which I fixed) where `horovod.keras.cross_rank()` was accidentally unavailable.

But those calls haven't been around for all that long, which I did not realize when I started using `hvd.cross_rank()` in our code.

Reply:
makes sense. disregard my suggestion then.